source/catalog.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Седона | Фото и видео</title>
+  </head>
+  <body class="page__body">
+    <header class="main-header">
+      <a class="main-header__logo" href="index.html">
+        <img src="img/city-of-sedona-logo.svg" width="185" height="96" alt="Логотип города Седона.">
+      </a>
+
+      <nav class="main-header__nav">
+        <button class="main-header__nav-toggle" type="button">
+          <span class="visually-hidden">Меню.</span>
+        </button>
+        <div class="nav-menu">
+          <ul class="nav-menu__list">
+            <li class="nav-menu__item">
+              <a class="nav-menu__link nav-menu__link--current" href="index.html">Главная</a>
+            </li>
+            <li class="nav-menu__item">
+              <a class="nav-menu__link" href="catalog.html">Фото и видео</a>
+            </li>
+            <li class="nav-menu__item">
+              <a class="nav-menu__link" href="form.html">Форма отзыва</a>
+            </li>
+            <li class="nav-menu__item">
+              <a class="nav-menu__link" href="https://htmlacademy.ru">HTML Academy</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main class="main-container">
+      <h1 class="visually-hidden">Фото и видео сайта туристического города Sedona.</h1>
+
+      <section class="background-image">
+        <h2 class="visually-hidden">Седона.</h2>
+        <img class="sedona-logo" src="img/sedona-logo.svg" width="229" height="39" alt="Седона.">
+      </section>
+
+      <section class="gallery">
+        <div class="gallery__heading page-article">
+          <h2 class="page-article__title">Фото и видео</h2>
+          <p class="page-article__text">Не можете решиться на путешествие из&#8209;за курса? Фотографии помогут вам забыть о политике и экономике.</p>
+        </div>
+        <ul class="gallery__items">
+          <li class="gallery-card">
```

Comment:
Давай обозначим, что card - это пункт списка, ну и чтобы было правильно
```suggestion
        <ul class="gallery__list">
          <li class="gallery__item gallery-card">
```